# Command the robot to navigate to a pose
geometry_msgs/PoseStamped target_base_pose
---
bool success
string message